charts/bitbucket-runner-autoscaler/templates/runnernamespace.yaml

```diff
@@ -1,6 +1,13 @@
+{{- $allNamespaces := (list) }}
+{{- range .Values.runner.config.groups }}
+  {{- $allNamespaces = append $allNamespaces .namespace }}
+{{- end }}
+{{- range $namespace := $allNamespaces | uniq }}
+---
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: bitbucket-runner
+  name: {{ $namespace }}
```

Comment:
Can we add a check that the namespace name is different from the controller namespace if this is a hard requirement?